packages/vue/tsconfig.json

```diff
@@ -1,12 +1,16 @@
 {
   "extends": "@aws-amplify/typescript-config/tsconfig.vue.json",
-  "include": [
-    "__mocks__",
-    "__tests__",
-    "*.ts",
-    "src/**/*.ts",
-    "src/**/*.tsx",
-    "src/**/*.vue"
-  ],
-  "exclude": ["node_modules"]
-}
+  "compilerOptions": {
+    "noImplicitAny": false,
+    "skipLibCheck": true,
+    "strictNullChecks": false,
+    "types": [
+      "node",
+      "jest",
+      "@testing-library/jest-dom",
+      "vite/client"
+    ]
+  },
+  "include": ["*.ts", "*.mjs", "scripts", "src/**/*.ts", "src/**/*.tsx", "src/**/*.vue"],
```

Comment:
Don't think there are any files with a _.tsx_ extension in this package?

Also why are we removing `"__mocks__"` and `"__tests__"` from the `includes` field?